src/feature/client/transports.c

```diff
@@ -909,6 +910,9 @@ handle_proxy_line(const char *line, managed_proxy_t *mp)
 
     parse_proxy_error(line);
     goto err;
+  } else if (!strcmpstart(line, PROTO_LOG)) {
```

Comment:
Hm. This approach here means that we can't have any command later that starts with LOG, like "LOGSETTING" or "LOGGING_OPTION or whatever.  Maybe we should check for "LOG " instead of "LOG"?

Reply:
Also what happens when the line is just "LOG" ?

Reply:
Fixed in 0b03646c8f5ba4093a720660dc83d9c2a6f4c4d7